k9mail/src/main/AndroidManifest.xml

```diff
@@ -247,6 +247,10 @@
                 <category android:name="android.intent.category.DEFAULT"/>
                 <category android:name="android.intent.category.BROWSABLE"/>
             </intent-filter>
+            <intent-filter>
+                <action android:name="${applicationId}.AUTOCRYPT_PEER_ACTION"/>
```

Comment:
What is this for? And why does the action name contain our applicationId?

Reply:
When listing identities of a key, OpenKeychain checks if this intent is registered, and if so makes the identity clickable. In our case, clicking the email address in a key will open the compose dialog then.

This was difficult before, because just launching a "write an email" intent didn't actually imply that the key would be used. This intent does, because it launches for the connected peer that we have a key for.